multicluster/main.go

```diff
@@ -161,7 +173,7 @@ func main() {
 		os.Exit(1)
 	}
 
-	setupLog.Info("starting manager")
+	setupLog.Info("starting modified manager")
```

Comment:
didn't get the point of adding`modified`.

Reply:
I was testing this, will remove it.